api/wd/index.md

```diff
@@ -826,3 +862,76 @@ browser
   .a11yAudit({ ignore: ['color-contrast', 'meta-viewport'] })
   .then(issues => deepEqual([], issues))
 ```
+
+### `assertAccessibilityScore(minScore: number[, skipAudits: Array<string>])`
+Runs Lighthouse 5.6 `accessibility` audit.
```

Comment:
Putting the version number everywhere feels like it's going to tie us to lots of updates every time the version changes.  Is it important to include?  If it's meaningful that it's the "new version" or something, could we instead say "Runs Lighthouse >= 5.6 ..." everywhere?